block/test/Test/Pos/Block/CborSpec.hs

```diff
@@ -67,10 +64,3 @@ spec = withGenesisSpec 0 defaultCoreConfiguration $ \_ ->
                     binaryTest @Block.SlogUndo
                     modifyMaxSuccess (min 50) $ do
                         binaryTest @Block.Undo
-                describe "Block network types" $ modifyMaxSuccess (min 10) $ do
-                    binaryTest @Core.MsgGetHeaders
-                    binaryTest @Core.MsgGetBlocks
-                    binaryTest @Core.MsgHeaders
-                    binaryTest @Core.MsgBlock
-                    binaryTest @Core.MsgStream
-                    binaryTest @Core.MsgStreamBlock
```

Comment:
These were moved up into the `lib` package because they need functionality provided by `infra`.